src/AmqpContext.php

```diff
@@ -89,4 +89,21 @@ public function iPublishInAmqpQueueMessageWithContent(string $transport, string
     {
         $this->adapter->publish($transport, $string, $command);
     }
+
+    /**
+     * @param string       $transport
+     * @param PyStringNode $string
+     *
+     * @throws \Exception
+     *
+     * @Then I acknowledge the content of next message in amqp queue :transport and its content is:
+     */
+    public function iAcknowledgeTheContentOfTheNextMessageInAmqpTransportAndItsContentIs(string $transport, PyStringNode $string): void
+    {
+        $messageContent = $this->adapter->acknowledgeAndGetNextMessageInTransport($transport);
+
+        if (json_decode($messageContent) != json_decode($string)) {
```

Comment:
In my opinion, we should not json_decode the content, the content of the rabbitmq message is may not in JSON.